### ISSUE TITLE
TypingError: Failed in nopython mode pipeline

### ISSUE THREAD

**Issue by Alalalalaki** [EconForge/interpolation.py#81]
I am trying out the dolo package. First it raise the jitclass problem described in #80. And I solve the problem by updating to the recent v2.2.0.

But then it raises a TypingError about `__eval_cubic` in `eval_splines.py`.

![image](https://user-images.githubusercontent.com/31154974/108640801-0546ff00-74df-11eb-9ecc-74ff5db8651e.png)

**Comment by albop** [comment]
What is your version of numba ? Maybe it doesn't support literal_value ?
Can you try to update numba ?

**Comment by Alalalalaki** [comment]
@albop I have numba v0.52.0. I think this is the latest version.

**Comment by albop** [comment]
Could you tell me more about what you are trying to do with dolo? Maybe
there is an incorrect call happening in dolo rather than interpolation.py ?

**Comment by Alalalalaki** [comment]
@albop I am running the example in "Getting started". This error is raised in the last procedure `simulate(model, dr) `.

```
from dolo import *                           # load the library
model = yaml_import("https://raw.githubusercontent.com/EconForge/dolo/master/examples/models/rbc.yaml")
                                             # import the model
display(model)                               # display the model
dr = time_iteration(model, verbose=True)     # solve
sim = simulate(model, dr)                    # simulate
```

**Comment by albop** [comment]
Thanks, I can reproduce it on my computer. It is indeed a regression in interpolation==2.2.0 . I'll fix it asap (this week, hopefully tomorrow). A quick fix is to install interpolation=2.1.6 and remove the line importing jitclass.

**Comment by albop** [comment]
Fixed by #83 . @Alalalalaki , you can try again if you wnat.
